monkey/monkey_island/cc/models/machine.py

```diff
@@ -12,12 +12,25 @@
 
 
 class Machine(MutableInfectionMonkeyBaseModel):
+    """Represents machines, VMs, or other network nodes discovered by Infection Monkey"""
+
     id: MachineID = Field(..., allow_mutation=False)
+    """Uniquely identifies the machine within the island"""
```

Comment:
This format is weird. Why not comment with # if it's a single line? Also, comments should be as close to commented code as possible, not they are closer visually to the line they don't comment

Reply:
See https://github.com/guardicore/monkey/pull/2228#discussion_r958241166